tests/integration/scenarios/dash_live.js

```diff
@@ -108,6 +108,7 @@ describe("dash live SegmentTimeline content", function () {
     expect(typeof videoRepresentation.index).to.equal("object");
 
     const videoRepresentationIndex = videoRepresentation.index;
+    const initVideoSegment = videoRepresentationIndex.getInitSegment();
 
     const initVideoSegment = videoRepresentationIndex.getInitSegment();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename "initVideoSegment" as this name is already used in declaration at line 111. ([javascript:S2814](https://sonarcloud.io/organizations/rx-player/rules#rule_key=javascript%3AS2814))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=249&issues=AWQT3116YwZU9vvrYAok&open=AWQT3116YwZU9vvrYAok)

[ISSUE_KEY]: AWQT3116YwZU9vvrYAok